### ISSUE TITLE
Exception: you forgot parenthesis in the command decorator for `subcommand`

### ISSUE THREAD

**Issue by rasa** [kdeldycke/click-extra#720]
With Python 3.10.11, and `click-8.1.6 click-extra-4.6.3 cloup-3.0.0` and running the script found at
https://kdeldycke.github.io/click-extra/parameters.html :
```python
#!/usr/bin/env python3

from click import option, echo, pass_context

from click_extra import config_option, extra_group

@extra_group
@option("--dummy-flag/--no-flag")
@option("--my-list", multiple=True)
@config_option
@pass_context
def my_cli(ctx, dummy_flag, my_list):
    echo(f"dummy_flag    is {dummy_flag!r}")
    echo(f"my_list       is {my_list!r}")
    echo(f"Raw parameters:            {ctx.meta.get('click_extra.raw_args', [])}")
    echo(f"Loaded, default values:    {ctx.default_map}")
    echo(f"Values passed to function: {ctx.params}")

@my_cli.command
@option("--int-param", type=int, default=10)
def subcommand(int_param):
    echo(f"int_parameter is {int_param!r}")
```
I get this error:
```text
Traceback (most recent call last):
  File "click_bug_test_example.py", line 23, in <module>
    def subcommand(int_param):
...
Exception: you forgot parenthesis in the command decorator for `subcommand`. While parenthesis are optional in Click >= 8.1, they are required in Cloup.
```
What am I doing wrong, or does this example code need freshening?

**Comment by rasa** [comment]
Fix is:
```
#!/usr/bin/env python3

from click import option, echo, pass_context

from click_extra import config_option, extra_group

@extra_group
@option("--dummy-flag/--no-flag")
@option("--my-list", multiple=True)
@config_option
@pass_context
def my_cli(ctx, dummy_flag, my_list):
    echo(f"dummy_flag    is {dummy_flag!r}")
    echo(f"my_list       is {my_list!r}")
    echo(f"Raw parameters:            {ctx.meta.get('click_extra.raw_args', [])}")
    echo(f"Loaded, default values:    {ctx.default_map}")
    echo(f"Values passed to function: {ctx.params}")

@my_cli.command() # added params
@option("--int-param", type=int, default=10)
def subcommand(int_param):
    echo(f"int_parameter is {int_param!r}")
```

**Comment by kdeldycke** [comment]
This has been fixed by #724
